### ISSUE TITLE
auth url incorrect

### ISSUE THREAD

**Issue by marcusrohden** [nhost/nhost-dart#108]
Hey guys, I'm having some issues trying to run my local environment. The host lookup has an extra dot ( . ) betweeen auth and nhost.

 **'local.auth..nhost.run'**

I'm currenly using the nhost_dart: ^1.0.0

Follow my NhostClient and exception log.

`return NhostClient(subdomain: Subdomain(subdomain: 'local', region: ''),`

`[log] Error trying to login: Failed host lookup: 'local.auth..nhost.run'`

**Comment by mhadaily** [comment]
Hi, 
Are you trying to use it localhost? in this case 

```
for local development, pass 'localhost' or 'localhost:1337' to the subdomain
```

**Comment by dbarrosop** [comment]
Solved in #119
